spec/unit/resources/splunk_service_spec.rb

```diff
@@ -1,137 +1,146 @@
 require_relative '../spec_helper'
 
-# TODO: Change how we're stubbing methods during refactoring so we can move this out of the global object space.
 include CernerSplunk::ServiceHelpers, CernerSplunk::RestartHelpers
```

Comment:
Some follow up to this,
I'm not totally concrete on this but I think stubbing things under Chef::Provider has issues, where Chef::Resource is fine with mixins in the describe block. That explains why it works in splunk_install_spec and not conf or service.

I'm actually surprised it works any differently at all, if it doesn't just outright not work. The block should be called at the top level due to the nature of blocks, but when mixing into the block, the libraries don't show up anywhere in the object space in RSpec. I haven't found any discussion or explanation of mixins in a block, probably because few people have been daring enough to try it.